### ISSUE TITLE
Export .map file

### ISSUE THREAD

**Issue by mateoconlechuga** [jacobly0/fasmg-ez80#3]
It would be great if ld.fasmg could export a .map file showing the different sizes and allocations of segements and functions used so that it could be used in CEmu, just like ZDS.

**Comment by mateoconlechuga** [comment]
```
Segment		Base	Top	Size
-------------	------	------	------
cleanup		D1A982	D1A995	14h
exit		D1A996	D1A99D	8h
libs		D1A996	D1A99D	8h
launcher	D1A882	D1A8D1	50h
startup		D1A947	D1A981	3Bh
bss		D031F6	D0343B	246h
code		D1A99E	D1C414	1A77h
data		D1C415	D1F239	2E25h

------
; labels
```

**Comment by jacobly0** [comment]
Fixed by 5f844562b799bb1b6712e6d3b069411b2d254bae.
